package-lock.json

```diff
@@ -1,5939 +1,5939 @@
 {
-    "name": "@mkkellogg/gaussian-splats-3d",
```

Comment:
please revert this change. It will render issues later on when we would like to rebase on the upstream